mountpoint-s3/src/cli.rs

```diff
@@ -325,6 +325,15 @@ pub struct CliArgs {
         value_name = "NETWORK_INTERFACE",
     )]
     pub bind: Option<Vec<String>>,
+
+    #[clap(
+        long,
+        help = "Set memory limit for CRT bufferpool, or 0 to use CRT default",
+        help_heading = CLIENT_OPTIONS_HEADER,
+        value_name = "MiB",
+        default_value = "0",
+    )]
+    pub crt_mem_limit: u64,
```

Comment:
Two thoughts:
- Can we call this `crt_mem_limt_mb` so the code is easier to read (I had to think about `1024*1024` below)
- Should this be an Option? I don't know if making "0" to mean the default is a great idea

Reply:
Yes, reasonable points, I will update the PR on both counts, thanks.